projects/safety_recipes/human_safety_evaluation/run.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import os
+from dataclasses import dataclass, field
+from typing import Any, List
+
+import hydra
+from mephisto.core.hydra_config import register_script_config
+from omegaconf import DictConfig
+
+from parlai.crowdsourcing.tasks.turn_annotations_static.turn_annotations_blueprint import (
+    STATIC_BLUEPRINT_TYPE,
+)
+from parlai.crowdsourcing.tasks.turn_annotations_static.util import run_static_task
+from parlai.crowdsourcing.utils.mturk import MTurkRunScriptConfig
+
+TASK_DIRECTORY = os.path.dirname(os.path.abspath(__file__))
+PARLAI_HOME = os.path.dirname(
+    os.path.dirname(os.path.dirname(os.path.dirname(os.path.realpath(__file__))))
+)
+BASE_TASK_DIRECTORY = os.path.join(
```

Comment:
you can define this more easily (and in a way that is less likely to break in the future if the paths change) by importing the `run.py` file from `turn_annotations_static` and then setting `BASE_TASK_DIRECTORY = os.path.dirname(os.path.abspath(run.__file__))`

Reply:
good pt!